src/components/molecules/EditRoomForm/EditRoomForm.tsx

```diff
@@ -24,6 +26,7 @@ interface EditRoomFormProps {
   onBackClick: (roomIndex: number) => void;
   onDelete?: () => void;
   onEdit?: () => void;
+  venueVisibility: RoomVisibility | undefined;
```

Comment:
In step with the other fields.

```suggestion
  venueVisibility?: RoomVisibility;
```